packages/date-picker/test/basic.test.js

```diff
@@ -1,17 +1,19 @@
 import { expect } from '@esm-bundle/chai';
-import { click, fixtureSync, keyboardEventFor, oneEvent, tap } from '@vaadin/testing-helpers';
+import { click, fixtureSync, keyboardEventFor, nextRender, oneEvent, tap } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import '../src/vaadin-date-picker.js';
 import { parseDate } from '../src/vaadin-date-picker-helper.js';
 import { close, open, touchTap, waitForOverlayRender } from './helpers.js';
 
 describe('basic features', () => {
-  let datePicker, input;
+  let datePicker, input, overlay;
 
-  beforeEach(() => {
+  beforeEach(async () => {
     datePicker = fixtureSync(`<vaadin-date-picker></vaadin-date-picker>`);
+    await nextRender();
```

Comment:
Added `nextRender()` here to make debugging easier and also to make Lit related changes easier.